### PR TITLE
Operations as a service

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
Terms of service

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-0430a52e`

File: docs/content/operations-as-service/offer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+`Version: 6 Nov 2022`
+
+# Operations and Maintenance as a Service
+
+In addition to the `Stakater App Agility Platform`, Stakater can provide operational and maintenance support for existing production clusters for flexible time periods. The Stakater Global SRE Team will be responsible for the operational efforts.
```
Comment (rasheedamir) [inline docs/content/operations-as-service/offer.md]: @karl-johan-grahn maybe we should remove the word production; as customer might want us to manage their non-prod clusters as well

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-b7a44b76`

File: docs/content/operations-as-service/sla.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## Availability Types
+
+To accurately identify types of degradations and the degree of their impact on the performance of services, we define several types of unavailability for which it is possible to set the parameters under the SLA.
+
+### Synthetic Availability
+
+Synthetic availability is the availability of running applications in the cluster and cluster components that are not related to the availability of the cluster's Control Plane, such as:
+
+* Applications that have been started before in containers that are still running
```
Comment (rasheedamir) [inline docs/content/operations-as-service/sla.md]: Need to discuss this text over call

File: docs/content/operations-as-service/sla.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+* Kubernetes services - the presence of corresponding `iptables` rules
+* DNS - the ability to resolve hosts
+
+The SLA for Synthetic Availability is valid only if these basic conditions are met:
+
+* Cluster nodes continue to be available
+* There is network connectivity between nodes
+* The application continues to operate correctly
```
Comment (rasheedamir) [inline docs/content/operations-as-service/sla.md]: We need to discuss this text over call

File: docs/content/operations-as-service/sla.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+### Control Plane Availability
+
+Control plane availability indicates that the deployment process in the cluster is working, and self-healing processes are running. It formulates the availability level of the main control OpenShift components:
+
+* `etcd`
+* `kube-apiserver`
+* `kube-controller-manager`
+* `kube-scheduler`
+* `kube-dns`
```
Comment (rasheedamir) [inline docs/content/operations-as-service/sla.md]: we should include the openshift-api-server and oauth server; plz check the names with genesis team

File: docs/content/operations-as-service/sla.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+* Vertical Pod Autoscaler
+* Horizontal Pod Autoscaler
+
+This availability type requires enabling the HA mode for components.
+
+### Extensions Availability
+
+Extensions availability means that additional Kubernetes components are available. It sets out the availability level of components not directly involved in delivering applications to the runtime environment and their operation in the cluster, such as:
+
+* Default monitoring stack
```
Comment (rasheedamir) [inline docs/content/operations-as-service/sla.md]: This seems to be covered in the previous availability type?

File: docs/content/operations-as-service/updating-clusters.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+## Types of Updates
+
+The following update types are available:
+
+* Minor-release is a planned update of cluster components that involves a feature extension or change. Releases result in incrementing the minor version (`Y`) component in the semantic version number (`X.Y.Z`).
+
+* Patch-release is an unplanned update of cluster components to fix errors or other functional issues that have already led or may lead to instability of the cluster. Patch-releases result in incrementing the patch version (`Z`) component in the semantic version number (`X.Y.Z`).
+
+The cluster may be updated on a schedule according to the update windows defined for the cluster. The available options for configuring update windows for the cluster are determined by the tariff in use, as shown in the table below:
```
Comment (rasheedamir) [inline docs/content/operations-as-service/updating-clusters.md]: @karl-johan-grahn is the word tariff correct here?

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-f0f60af6`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-db0bac31`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-8bd8a1c6`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-70-44b45824`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.45`

**Pull request merged**